src/cssUsage.js

```diff
@@ -605,69 +605,82 @@ void function() { try {
+
+                        // Remove group contents (...), {...} and [...]
+                        value = value.replace(/[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]*)[)])*[)])*[)])*[)])*[)]/g, " <parentheses-block> ");
+                        value = value.replace(/[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]*)[)])*[)])*[)])*[)])*[)]/g, " <parentheses-block> ");
+                        value = value.replace(/[{](?:[^{}]+|[{](?:[^{}]+|[{](?:[^{}]+|[{](?:[^{}]+|[{](?:[^{}]*)[}])*[}])*[}])*[}])*[}]/g, " <curly-brackets-block> ");
+                        value = value.replace(/[{](?:[^{}]+|[{](?:[^{}]+|[{](?:[^{}]+|[{](?:[^{}]+|[{](?:[^{}]*)[}])*[}])*[}])*[}])*[}]/g, " <curly-brackets-block> ");
+                        value = value.replace(/[\[](?:[^\[\]]+|[\[](?:[^\[\]]+|[\[](?:[^\[\]]+|[\[](?:[^\[\]]+|[\[](?:[^\[\]]*)[\]])*[\]])*[\]])*[\]])*[\]]/g, " <square-brackets-block> ");
+                        value = value.replace(/[\[](?:[^\[\]]+|[\[](?:[^\[\]]+|[\[](?:[^\[\]]+|[\[](?:[^\[\]]+|[\[](?:[^\[\]]*)[\]])*[\]])*[\]])*[\]])*[\]]/g, " <square-brackets-block> ");
+
+                        break;
+
+                    default:
+
+                        // Replace strings by dummies
+                        value = value.replace(/"([^"\\]|\\[^"\\]|\\\\|\\")*"/g,' <string> ')
+                            .replace(/'([^'\\]|\\[^'\\]|\\\\|\\')*'/g,' <string> ');
+
+                        // Replace url(...) functions by dummies
+                        if (value.indexOf("(") != -1) {
+                            value = value.replace(/([a-z]?)[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]*)[)])*[)])*[)])*[)])*[)]/g, "$1() ");
+                            value = value.replace(/([a-z]?)[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]+|[(](?:[^()]*)[)])*[)])*[)])*[)])*[)]/g, "$1() ");
+                        }
+
+                }
+            }
+            else {
+                switch (propertyName) {
+                    // If a URL value is given for the cursor property, then we want to remove all the speech marks put in around the url links
+                    // to prevent inconsistencies in results (in some running cases they are put in, others not) and split on the fallback values supplied.
+                    case 'cursor':
+
+                        // Remove various quotes - Crawler has some issue relating to speech marks, where sometimes they are put around url links (local test page run) and sometimes not (crawler run).
+                        if (value.indexOf("'") != -1 || value.indexOf("‘") != -1 || value.indexOf('"')) {
+                            value = value.replace(/('|‘|’|")/g, "");
+                        }
+
+                        // Divide at commas to separate cursor url value and supplied fallback values.
+                        value = value.split(/\s*,\s*/g);

```

Comment:
This should be done after the white-space trimming and collapsing otherwise the entries will start or end with spaces (for the first and last ones)

Reply:
I don't want to do whitespace trimming for the cursor special case.